_docs/principles/accessibility.md

```diff
@@ -198,61 +303,71 @@ Helix minimum spacing standards require: 
 
-#### Failed text inputs
+</section>
 
-There are many acceptable ways to make text inputs visually accessible. If you're indicating a failed text input, make sure to include multiple visual indicators. For example, in addition to the error message, the border color turns red to draw the eye and doubles in thickness from 1px to 2px to provide another distinguishable visual cue that is not reliant on color.
+<section class="static-section"  markdown="1">
 
-It is important to reserve the prominent border at 2px for failed text input as an additional visual indicator for error an state.
+<div class="hxRow"  markdown="1">
 
-{% endcolumn %}
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+## Interactivity guidelines
 
-{% figure [caption:"Default state"] [class:"image bg-light border"] %}
-<embed src="{{site.url}}/assets/images/overview/accessibility/accessibility-inputs-1.svg"/>
-{% endfigure %}
+Consider the following page layout guidelines when you construct an accessible UI.
 
-{% endcolumn %}
+### Ensure that pointer targets are large enough
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+Pointer targets that are activated by a mouse, trackball, or stylus should be at least 44px x 44px. Large pointer targets can be helpful to users who have difficulty with fine motor skills.
 
-{% figure [caption:"Error state"] [class:"image bg-light border"] %}
-<embed src="{{site.url}}/assets/images/overview/accessibility/accessibility-inputs-2.svg"/>
-{% endfigure %}
+### Avoid revealing content on hover
 
-{% endcolumn %}
+Users that rely on screen readers or a keyboard to navigate the control panel cannot interact with elements that can only be revealed by using a mouse.
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+### Write meaningful alternative text for images
 
-#### Focus for Input Field
+Screen readers read the text you enter in the `alt` attribute that you associate with an image. If an image provides important contextual information, then use the `alt` attribute to describe what is happening in the picture. If you do not add an `<alt>` tag, some screen readers read the name of the image file, which introduces unnecessary confusion and results in a poor user experience.
 
-In this example, we provide a clear visual feedback that the focus has moved to the input field by changing the border color and adding a highlight to give the user clear indication of where they’re currently taking action.
+If an image is purely decorative, use `aria-hidden="true"`. Do not use an empty `alt` attribute.
+
+The `aria-hidden="true"` also applies to SVG inline images such as icons. Inline images require you to add other elements, for example `<title>` and `<description>`, to make them accessible. For more information about SVGs, see [Accessible SVGs](https://css-tricks.com/accessible-svgs/).
```

Comment:
`Inline images` -> `Inline SVGs` (`<title>` and `<description>` don't work for `<img>` elements)

Reply:
Done.